code/modules/explorer_drone/loot.dm

```diff
@@ -152,7 +152,6 @@ GLOBAL_LIST_INIT(adventure_loot_generator_index,generate_generator_index())
 	lefthand_file = 'icons/mob/inhands/items/firelance_lefthand.dmi'
 	var/windup_time = 10 SECONDS
 	var/melt_range = 3
-	var/charge_per_use = 200
```

Comment:
Why remove the variable?

Reply:
It was used only in 1 place. Waste of a variable

Reply:
Could be a define, but ideally we shouldn't be using magic numbers.